R/RLearner_fcregr_Arima.R

```diff
@@ -0,0 +1,91 @@
+#'@export
+makeRLearner.fcregr.Arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.Arima",
+    package = "forecast",
+    par.set = makeParamSet(
```

Comment:
Agree, I removed the shorter line breaks, but did keep it for some when the lines became unreasonably long